src/cowboy_clock.erl

```diff
@@ -83,10 +83,12 @@ handle_cast(_Msg, State) ->
 	{noreply, State}.
 
 -spec handle_info(any(), State) -> {noreply, State} when State::#state{}.
-handle_info(update, #state{universaltime=Prev, rfc1123=B1, tref=TRef}) ->
+handle_info(update, #state{universaltime=Prev, rfc1123=B1, tref=TRef0}) ->
+	erlang:cancel_timer(TRef0),
```

Comment:
Yep. Then I'm fine with it but would like a comment above the send_after like

```
%% Cancel the timer in case an external process sent an update message.
```

Otherwise I risk removing it by mistake in the future.